backend/runners/load_representative_points.py

```diff
@@ -159,5 +159,10 @@ def _get_all_service_areas(features):
     return service_areas
 
 
+def _sanitize_county_name(county_str):
+    """Clean a county string for use in the service_area_id field."""
+    return county_str.lower().replace(' ', '_').replace('.', '')
```

Comment:
Can you have this function take a map of sanitization rules maybe?

Reply:
Also, are we fine with this fix of removing `.` @viviandien ?

Reply:
This is only in the `service_area_id` field. Let me clarify that in the code.